Library/Formula/dnscrypt-proxy.rb

```diff
@@ -53,6 +79,9 @@ def caveats; <<-EOS.undent
     resolver. If you would like to change these settings, you will have to edit
     the plist file (e.g., --resolver-address, --provider-name, --provider-key, etc.)
 
+    If at some point the resolver file gets outdated, it can be updated to the
+    latest version by running: #{libexec}/update-resolvers
```

Comment:
Looks like this should be `#{libexec}/update-resolvers.sh`. Might be nice to name this `dnscrypt-update-resolvers` and but it in `bin`?